content/applications/websites/ecommerce/products/cross_upselling.rst

```diff
@@ -2,88 +2,65 @@
+**Accessory products** are showcased in the :guilabel:`Suggested Accessories` section during the
+:guilabel:`Review Order` step, just before proceeding to checkout.
 
 To enable **accessory products**, go to :menuselection:`Website --> eCommerce --> Products`, select
 a product, go to the :guilabel:`Sales` tab, and enter the products you wish to feature in the
-:guilabel:`Accessory Products` field. Suggested accessory products are **linked** to the product(s)
-they are set up with on the **product template**. They only appear when that product is at the
-checkout process review.
+:guilabel:`Accessory Products` field.
 
-Upselling
-=========
+.. image:: cross_upselling/accessory-products.png
+   :alt: Suggested accessories at checkout during cart review
 
-.. _cross_upselling/alternative:
+.. _ecommerce/cross_upselling/alternative:
 
 Alternative products
---------------------
+====================
 
-**Alternative products** are suggested on the **product page** and usually incentivize customers to
-buy a more expensive variant or product than the one they were initially shopping for.
+You can display alternative products at the bottom of the product page as an **upselling
+technique**, encouraging customers to consider a more expensive variant or product than their
+initial choice by presenting appealing alternatives.
 
 .. image:: cross_upselling/cross_upselling-alternative.png
-   :align: center
    :alt: Alternative products on the product page
 
-To enable **alternative products**, go to :menuselection:`Website --> eCommerce --> Products`,
-select a product, go to the :guilabel:`Sales` tab, and enter the products you wish to feature in the
-:guilabel:`Alternative Products` field. Then, go to the related **product page** by clicking
-:guilabel:`Go To Website`, and click :menuselection:`Edit`. Stay on the :guilabel:`Blocks` tab, and
-scroll down to the :guilabel:`Dynamic Content` section. Then, drag and drop the :guilabel:`Products`
-building block anywhere on the **product page**.
-
-When placed, in :guilabel:`Edit` mode, click the **block** to access various settings for that
-:guilabel:`Products` building block. In the :guilabel:`Filter` field, select :guilabel:`Alternative
-Products`. You can configure several additional settings, such as how many elements are displayed
-(:guilabel:`Fetched Elements`), the :guilabel:`Template` used, etc.
+To enable alternative products, navigate to :menuselection:`Website --> eCommerce --> Products`,
+select a product, go to the :guilabel:`Sales` tab, and in the :guilabel:`Alternative Products`
+field, enter the products you want to feature.
+
+.. tip::
+
+   You can then adjust various settings, such as :guilabel:`Template`, :guilabel:`Fetched Elements`
+   (number of items displayed), and :guilabel:`Slider Speed`. To do so, click
+   :guilabel:`Go To Website` to visit the related product page, select the block showing the
+   alternative products, and go to the :guilabel:`Products` section.
```

Comment:
I think this is confusing because:
- The context is missing (+ transition from the text above, like I mentioned in my previous review): it's not clear what we're talking about (i.e., the Alternative products *building block*);
- You mention the settings (and related field names) you can adjust *before* explaining how to access those settings. For longer instructions like these, respecting the order of things (as they appear on the screen) really helps make the instructions clearer.
- We're missing some info about clicking Edit and selecting the building block (otherwise people will not find the settings you mentioned).